### DISCUSSION TITLE
Support generating new wallet addresses from CLI

### DISCUSSION THREAD

**Discussion by mercurato** [Chia-Network/chia-blockchain#8082] [Ideas]
Currently you cannot generate a new wallet address from the CLI and it can only be accessed from the GUI
An extra command line argument like --generate after chia wallet get_address would be an easy addition

**Comment by BrandtH22** [comment]
@mercurato , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the flag `--new-address` can be used with the get_address command to generate a new address.
https://docs.chia.net/wallet-cli#get_address

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
